README.md

```diff
@@ -23,7 +23,8 @@ fit_dihedral_with_gomc(
         atom_type_naming_style='general',
         gomc_cpu_cores=1,
         r_squared_min=0.99,
-        r_squared_rtol=1e-03
+        r_squared_rtol=1e-03,
+        opls_set_k0_zero=True
```

Comment:
```suggestion
        opls_force_k0_zero=True
```

Reply:
or we can change the rest to `set` (tbh I like `set` better)

Reply:
I had that but I changed to force.  I thought force was stronger because it does not allow for later shifting that is standard

Reply:
then `force` sounds reasonable then

Reply:
discussing with everyone, I think we will leave at `opls_force_k0_zero`